modules/openapi-generator/src/main/resources/go/api.mustache

```diff
@@ -299,7 +299,7 @@ func (a *{{{classname}}}Service) {{{nickname}}}(ctx context.Context{{#hasParams}
 			}
 			newErr.model = v
 			return {{#returnType}}localVarReturnValue, {{/returnType}}localVarHttpResponse, newErr
-		}
+		{{^wildcard}} } {{/wildcard}}
 		{{/dataType}}
 		{{/responses}}
 		return {{#returnType}}localVarReturnValue, {{/returnType}}localVarHttpResponse, newErr
```

Comment:
This gets generated even when there is a default response. We end up with a double return in the final code. But I couldn't find a way to reliably avoid it. Any ideas?